code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -128,33 +125,35 @@
 		else if(tamed == 1 && isrobot(found_atom))
 			return null
 		else
+			if(resting)
+				lay_down()
 			return found_atom
 	else
 		return null
 
 /mob/living/simple_animal/otie/friendly/security/Found(var/atom/found_atom)
-	if(!SA_attackable(found_atom))
-		return null
-	if(istype(found_atom,/mob/living/simple_animal/mouse))
-		return found_atom
 	if(check_threat(found_atom) >= 4)
+		if(resting)
+			lay_down()
 		return found_atom
-	if(found_atom in friends)
```

Comment:
You're on the right track by removing redundant shit like this, but this was only the 3rd layer. That's how much of a mess Oties are.